JASP-Engine/JASP/R/correlation.R

```diff
@@ -401,7 +401,7 @@ Correlation <- function(jaspResults, dataset, options){
       }
       
       result$vsmpr <- JASP:::.VovkSellkeMPR(result$p.value)
-      result$vsmpr <- ifelse(result$vsmpr == "∞", Inf, result$vsmpr)
+      result$vsmpr <- ifelse(result$vsmpr == "\u221E", Inf, result$vsmpr)
```

Comment:
Did you test this??
I have tried, but I cannot get  a situation where this vsmpr  becomes infinite.

Reply:
No I didn't, but I looked up the codepoint for infinity and it is that: https://codepoints.net/U+221E

Reply:
This line can just be removed. In https://github.com/TimKDJ/jasp-desktop/commit/9a821494b6deda4800ea8f4538701d88c279007f# I changed the `.VovkSellkeMPR()` function so it returns a numeric value rather than a character.